bokehjs/src/lib/core/view.ts

```diff
@@ -129,6 +129,29 @@ export class View implements ISignalable {
     }
   }
 
+  on_transitive_change<T>(property: Property<T>, collect: (value: T) => HasProps[], fn: () => void): void {
```

Comment:
This watches for changes on child models?

Reply:
It watches for models provided by `collect()`, which allows working with a subset of child models (for this property) and doesn't require generalized traversal of data structures.

Reply:
Actually, I dropped `collect()` in favour of `HasProps.references()`, which greatly simplifies usage.